authentication/AuthenticationExample/View Controllers/Other Auth Method Controllers/PhoneAuthViewController.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import Firebase
+
+
+class PhoneAuthViewController: OtherAuthViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI(for: .PhoneNumber)
+    }
+    
+    override func buttonTapped() {
+        guard let phoneNumber = textField.text, !phoneNumber.isEmpty else { return }
+        phoneAuthLogin(phoneNumber)
```

Comment:
So currently, the keyboard type is only numbers for this textfield. There is also a label underneath that gives an example input.

Reply:
I found some really cool swift regex solutions for providing format to the textfield to look like a formatted phone number  while you type. Something like this might be worth for providing?

Reply:
I think leaving it simple is probably fine without the complex regex just in case... one thing to consider is an external keyboard can type characters not visible on the on screen keyboard. Syntax is fine, it was always just a pet peeve of mine when adding something like my postal code as `A1A 1A1` and it throwing an error saying "your format is invalid, I want `A1A1A1`".

Leaving the instructions as is is fine, and we can leave it as is.

Reply:
did not think of that part. but ok, SG!